### PR TITLE
enforce formatting according to .editorconfig

### PR DESCRIPTION
**Pull request by fbartels**
relates to #1221

### CODE REVIEW

**Comment by fbartels** [comment]
@erik-wramner as you already thought this chance is quite invasive (and current prs will probably need to be rebased and merge conflicts resolved).

@j-marz i've seen you started work on a shellcheck pr. what are your thoughts towards this?

**Comment by erik-wramner** [comment]
Yes, it affects a lot of code. Looking at the active PRs we have:

- ShellCheck - needs to rebase/merge or commit first
- Modularize tests - needs to rebase/merge or commit first
- Port descriptions - just docs, not affected
- SA short circuit - waiting for action, stale? Can probably close.
- Regex aliases - stale, will close Tomorrow unless I get an answer
- Sqlite - I think we should reject that as outside the project scope
- Check for updates of certs - I think we should reject that in favor of frequent updates
- Test separation - stale, kept for reference, no need to rebase
- Change pw - should ideally use other container? Relatively free-standing
- Dovecot mysql - I think we should reject that as outside the project scope
- New Debian version (branch) - not sure of impact but would need to rebase/merge

Not that bad except for the new Debian version. I don't know how many changes Tom has made to the code there. It would be nice to get some input from @tomav on this.

File: .editorconfig [hunk 1]
```diff
@@ -27,6 +26,10 @@ indent_size = 2
 indent_style = space
 indent_size = 2
 
-[target/start-mailserver.sh]
```
Comment (erik-wramner) [inline .editorconfig]: What happens with start-mailserver.sh? Do we use 2 spaces now, or is it not checked at all?

File: config/postfix-accounts.cf [hunk 2]
```diff
@@ -0,0 +1 @@
+# TODO is this file even used?
```
Comment (erik-wramner) [inline config/postfix-accounts.cf]: I think this and the other similar files are placeholders that make it easy for new user to find where the can make changes. The .bak file has probably been checked in by mistake though.

File: .editorconfig (hunk 1 above)
Comment (fbartels) [reply .editorconfig]: With this change `start-mailserver.sh` is simply treated like all the other bash scripts in this repository.  But even with the exception for start-mailserver there were quiet a lot of lines that would have needed fixing (did not count).

File: config/postfix-accounts.cf (hunk 2 above)
Comment (fbartels) [reply config/postfix-accounts.cf]: It feel a bit like this is a relic of a change of the past. I did not immediately find where this file is put into the container and therefore decided to add this todo.

File: .editorconfig (hunk 1 above)
Comment (erik-wramner) [reply .editorconfig]: Yes, I've noticed.

**Pull request closed without merging**